src/node.cc

```diff
@@ -452,9 +456,25 @@ void LoadEnvironment(Environment* env) {
 
 
 #ifdef __POSIX__
+
+
+static sa_handler previous_sigsegv_handler = nullptr;
```

Comment:
apparently `sa_handler` isn't a type one can use here, what should i use instead?

Reply:
`typedef void (*sa_handler)(int); static sa_handler previous;`? I'm not sure nullptr will assign to it, you'll see. Or, do `struct sigaction previous_action;`, and assign to/read from `previous_action.sa_handler`, then you don't need any new types.

Reply:
@sam-github thank you for your insight!